app/components/dashboard/hypertension/overdue_patients_return_to_care_component.rb

```diff
@@ -0,0 +1,52 @@
+class Dashboard::Hypertension::OverduePatientsReturnToCareComponent < ApplicationComponent
+  attr_reader :data
+
+  def initialize(region:, data:, period:, with_removed_from_overdue_list:)
+    @region = region
+    @data = data
+    @period = period
+    @contactable = !with_removed_from_overdue_list
+  end
+
+  def graph_data
+    if @contactable
```

Comment:
For readability, can be changed to 
```ruby
if @contactable
  {}
else
  {}
end
```

Reply:
also, we're using/not-using `@` interchangeably. Either refer to contactable & data with @ or without.